tests/test_sinogram.py

```diff
@@ -1,14 +1,16 @@
 import numpy as np
 import pytest
 from skimage import data
-from skimage.transform import radon
+from skimage.transform import iradon, radon
 
 from aspire.image import Image
 from aspire.utils import grid_2d
 
 # Relative tolerance comparing line projections to scikit
-# The same tolerance will be used in all scikit comparisons
-SK_TOL = 0.005
+# The same tolerance will be used in all scikit forward and backward comparisons
+SK_TOL_FORWARDPROJECT = 0.005
+
+SK_TOL_BACKPROJECT = 0.2
```

Comment:
This tolerance seems high compared to the forward direction. It also looks like our reconstructed images do not match scikit's reconstructions visually when I plot them. I wonder if we have a bug or if this is expected.

Reply:
With some fixes we were able to reduce this down to 0.0025.

Reply:
That's great!